sources/src/main/java/com/ullink/slack/simpleslackapi/SlackSession.java

```diff
@@ -51,6 +50,8 @@
 
     SlackMessageHandle<SlackMessageReply> sendMessage(SlackChannel channel, SlackPreparedMessage preparedMessage, SlackChatConfiguration chatConfiguration);
 
+    SlackMessageHandle<SlackMessageReply> sendMessage(SlackChannel channel, JsonObject message, SlackChatConfiguration chatConfiguration);
```

Comment:
I don't think you should expose the underlying JSON library used, instead if you want to use ephemereal messages you could set a new boolean property into the SlackChatConfiguration class. Then in the implementation of this method you should check for this property value and add the json parameter accordingly like it is done here : https://github.com/Ullink/simple-slack-api/blob/master/sources/src/main/java/com/ullink/slack/simpleslackapi/impl/SlackWebSocketSessionImpl.java#L494